include/networkit/viz/GraphLayoutAlgorithm.hpp

```diff
@@ -124,18 +126,19 @@ class GraphLayoutAlgorithm {
         file << "@balllist {a} color= blue master={points} radius= 0.05" << std::endl;
 
         G->forNodes([&](node u) {
-            file << "{a}" << vertexCoordinates[u][0] << " " << vertexCoordinates[u][1] << " " << vertexCoordinates[u][2] << std::endl;
+            file << "{a}" << vertexCoordinates[u][0] << " " << vertexCoordinates[u][1] << " "
+                 << vertexCoordinates[u][2] << std::endl;
         });
 
         // edges
         file << std::endl;
         file << "@subgroup {edges} dominant" << std::endl;
         file << "@vectorlist {edges} color= white" << std::endl;
         G->forEdges([&](node u, node v) {
-            //if (u <= v) { // draw graph undirected
```

Comment:
Does the formatting automatically remove the comment here? In this case it more likely an artifact (so it's fine I think), but in general deleting comments by formatting should be avoided.

Reply:
No, I thought it was ok to remove unused code so I did it, but I can move this change to another PR if necessary.